src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -316,7 +695,14 @@ public void Save(ModelSaveContext ctx)
             {
                 ctx.SaveBinaryStream("TFSavedModel", w =>
                 {
-                    string[] modelFilePaths = Directory.GetFiles(_savedModelPath, "*", SearchOption.AllDirectories);
+                    // only these files need to be saved.
+                    string[] modelFilePaths =
```

Comment:
>modelFilePaths [](start = 29, length = 14)

Is this a change in the serialization format, or just a more explicit way of writing it?

Reply:
No change in the format. Previously, our assumption was that folder will contain only these files so were writing all files in the folder. Now folder may contain archive files due retraining. So, I am explicitly writing required files.

---
In reply to: [222727569](https://github.com/dotnet/machinelearning/pull/1063#discussion_r222727569) [](ancestors = 222727569)